contents/handbook/people/hiring-process/engineering-hiring.md

```diff
@@ -46,11 +46,11 @@ For full-stack roles, the task involves building a small web service (both backe
 
 An engineering SuperDay usually looks like this (_there is a degree of flexibility due to time zone differences)_:
 
-*   An invitation to a personal Slack channel for your SuperDay, which we'll use throughout the day
+*   An invitation to a personal Slack channel for the SuperDay, which we'll use throughout the day
 *   Kick-off session with an engineer
 *   Time to focus on the task
 *   A "peer interview" with a couple of members of our team, so that both us and the candidate can see if we're a fit
-*   A chat with [James](/james) (or [Tim](/tim), if you met with James in the previous stage)
+*   A chat with [James](/james) or [Tim](/tim), whoever they didn't meet with in the previous stage
```

Comment:
This isn't always the case so maybe put sometimes?

Reply:
I think it almost always happens?

Reply:
If they are off or travelling it won't. Like my Superday today doesn't have it.

Reply:
Yeah it should happen except in very exceptional circumstances (I guess you got, um, special treatment? 😅)